config/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,2 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.tapptitude.config">
-
-</manifest>
\ No newline at end of file
+<manifest />
```

Comment:
packaged move to `build.gradle.kts` `namespace` property.

Reply:
I think we can remove these empty manifest files 🤔